Application/EdFi.Ods.Features/OpenApiMetadata/Factories/OpenApiMetadataPathsFactory.cs

```diff
@@ -179,11 +180,14 @@ private Operation CreateGetByIdOperation(OpenApiMetadataPathsResource openApiMet
             {
                 parameters.Add(new Parameter
                 {
-                    name = "Snapshot-Identifier",
+                    name = "Use-Snapshot",
                     @in = "header",
-                    description = "Indicates the Snapshot-Identifier that should be used.",
+                    description = "Indicates if the configured Snapshot should be used.",
                     type = "string",
-                    required = false
+                    required = false,
+                    @enum = System.Enum.GetNames<SnapshotUsage>(),
```

Comment:
I'm thinking that we don't want to use the `On`/`Off` enumeration values for parsing the `Use-Snapshot` header value. The header should be a simple `true`/`false` value, which is more intuitive for API developers. We need the typed enumeration only for use internally with the `IContextProvider<T>` pattern.